### PR TITLE
Register os.is-a.dev

### PR DESCRIPTION
**Pull request by heyturkiye204**
Register os.is-a.dev with CNAME record pointing to cdn-rsdv-hey.netlify.app

### CODE REVIEW

**Review by iostpa — CHANGES_REQUESTED** [review]
I don't think we allow www.subdomain (could be wrong though)

File: domains/www.heytr.json
Comment (CuteDog5695) [inline domains/www.heytr.json:1]: Delete this file.

File: domains/os.json
Comment (CuteDog5695) [inline domains/os.json:1]: Just a comment, but im not sure if the name is allowed (though im thinking like this since im a bit picky)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
I guess its fine now

**Pull request merged**